packages/sync/src/class-defaults.php

```diff
@@ -894,7 +894,7 @@ public static function is_whitelisted_option( $option ) {
 		'promote_users',
 		'delete_themes',
 		'export',
-		'edit_comment',
+		'moderate_comments',
```

Comment:
This is now added twice, see line 854

Reply:
Fixed in c0f0feda6fd3b47d17571dc9326f83170e0eb21d